elastic/apm-server
# Add service name to dataset field

## Motivation/summary

Implement the indexing strategy for APM data

## Checklist

## How to test these changes

Install APM package, send data to apm-server, and observe that data streams are created containing the ingested service names, and no legacy indices are created

## Related issues

Closes https://github.com/elastic/apm-server/issues/4492
Requires https://github.com/elastic/kibana/pull/89870
